doc/vstgui-dev.md

```diff
@@ -0,0 +1,60 @@
+# Developing with our VSTGUI Fork
+
+We made the hard choice [here](https://github.com/surge-synthesizer/surge/issues/515) to fork
+vstgui so we can maintain our own fixes while also submitting them to steinberg
+
+## How it is set up
+
+* `surge-synthesizer/vstgui` is a form of `steinbergmedia/vstgui` with default branch set to master
```

Comment:
this thing does not exist.
you mean `surge-synthesizer/vst3sdk/vstgui` ?

Reply:
I mean https://github.com/surge-synthesizer/vstgui

I should just put that url in the doc

Reply:
And form should say fork

This was hard for me to describe in doc. Valuable to see if any of the doc makes sense. Thank you

Reply:
Let me push some cleaner doc up. Gimme 10 minutes.